### PR TITLE
Images in nav

### PR DESCRIPTION
**Pull request by davidpmccormick**
References #1372

## Type
✨ Feature  

- [x] Demoed to @jennpb 

## Value
Puts image search in the main nav.

## Screenshot
screen shot 2017-08-24 at 16 40 39

### All
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### CODE REVIEW

File: server/views/layout/default.njk [hunk 1]
```diff
@@ -50,6 +50,11 @@
         isCurrent: pageConfig.inSection === 'explore'
       },
       {
+        href: "/search",
```
Comment (jamesgorrie) [inline server/views/layout/default.njk]: Could we make this `/works`

**Comment by jamesgorrie** [comment]
@jennpb are we planning on adding this to v1?

**Comment by jennpb** [comment]
I think we should if we can. We'll need clear signposting for anyone coming directly to the site.

**Comment by jamesgorrie** [comment]
> if we can

We can

**Pull request merged**